### PR TITLE
PST import caveat with autoexpansion

### PR DESCRIPTION
**Pull request by v-padum**
Adding caveat for network import into mailboxes that have autoexpanding turned on. (Premier support ticket 32045647)

### CODE REVIEW

File: microsoft-365/compliance/enable-autoexpanding-archiving.md [hunk 1]
```diff
@@ -36,6 +36,8 @@ You can use the Exchange Online auto-expanding archiving feature to enable addit
...
         
         If auto-expanding archiving is enabled on an inactive mailbox, the only way to recover data is by using the Content search tool in the Microsoft Purview compliance portal to export the data from the mailbox and import to another mailbox. For more information, see the [Inactive mailboxes and auto-expanding archives](inactive-mailboxes-in-office-365.md#inactive-mailboxes-and-auto-expanding-archives).
+        
+    - Once auto-expanding archiving is enabled on a mailbox, [network upload of PST files](https://docs.microsoft.com/en-us/microsoft-365/compliance/use-network-upload-to-import-pst-files?view=o365-worldwide) may fail. This is due to the sharding of specific folders into auxiliary archives, which causes imports to a sharded location to generate an error.
```
Comment (JohanFreelancer9) [inline microsoft-365/compliance/enable-autoexpanding-archiving.md:40]:
```suggestion
    - Once auto-expanding archiving is enabled on a mailbox, [network upload of PST files](/microsoft-365/compliance/use-network-upload-to-import-pst-files) may fail. This is due to the sharding of specific folders into auxiliary archives, which causes imports to a sharded location to generate an error.
```

**Review by JohanFreelancer9 — CHANGES_REQUESTED** [review]
Copy review - @v-padum, kindly commit the suggested changes. Many thanks.

**Comment by v-padum** [comment]
Yep perfect and thank you, wasn't sure how to make the relative URL work here given that every markdown file is in a different location.

**Review by JohanFreelancer9 — APPROVED** [review]

**Comment by cabailey** [comment]
#hold-off Discussing internally

**Comment by yogkumgit** [comment]
@cabailey Any update please.

**Comment by v-padum** [comment]
Hi Yogesh. I believe we're cancelling this PR as document owners would want the information and context being added to be in a different location.

Thanks.

**Comment by cabailey** [comment]
Yes, we agreed to move the support statement for PST files from the docs for auto-expanding to the PST import docs (with any new caveats) & I left this PR open in case you wanted to use this for the update & tracking.

**Comment by cabailey** [comment]
Closing.  We agreed offline that support statements for PST import belong with the PST import docs.

**Pull request closed without merging**